docs/conf.py

```diff
@@ -75,8 +75,6 @@ def _load_data(self, env, data_source, encoding):
     'videos/prague/2018/tackling-technical-debt-in-the-docs-louise-fahey.rst',
 ]
 
-html4_writer = True
```

Comment:
This seems unrelated? I'm :+1: on this change without this here.

Reply:
Check! Pipeline was failing before i believe, that's why i ran into this. Resolved!